requirements.txt

```
@@ -12,4 +12,4 @@ google-cloud-spanner==3.42.0
 psutil
 
 # To run SLEAP process
-sleap==1.3.3
\ No newline at end of file
+sleap[pypi]==1.3.4
```

Nit, add newline to end of file
```suggestion
sleap[pypi]==1.3.4

```